src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml

```diff
@@ -65,7 +65,7 @@ tests:
     - label:
           "Given DGTHREAD.S.F01(ERRCNT) ensure featuremap has the correct bit
           set"
-      PICS: DGETH.S.F01
+      PICS: DGTHREAD.S.F01
```

Comment:
`DGTHREAD.S.F01` does not exist in PICS v0.11.

Reply:
@yinyihu-silabs Script is as per testplan to use PICS: DGTHREAD.S.F01 in step 3.
![image](https://user-images.githubusercontent.com/102359958/188702027-57dad48a-6e3f-4a63-acc8-145cfacc6bf6.png)

Reply:
@manjunath-grl 
PICS xml 0.11 does not have the same PICS above
![image](https://user-images.githubusercontent.com/83772323/188702451-24586aa4-97b3-4a75-af2d-3c007abafc5a.png)